### DISCUSSION TITLE
what's the right way to powerwash my chromebook

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#832] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so im not sure how, I'm deciding to use a 1500 psi pressure washer or a 2000 psi pressure washer. the 1500 has less flow but is cheaper and more reliable. could someone help me on this topic? thanks so much!

**Comment by Su4o** [comment, marked as answer]
Yes those are all very good options, but you should probably use the "hand" option instead (free, but not that fast!) Here are the steps to powerwash the chromebooK using the hand option.

1. Turn of your chromebook. (this step is important to the process of this procedure.)
2. Flip the chromebook and remove the screws using a screwdriver. 
3. Use the "pencil bypass" written in coolelectronic's writeup blog. (https://blog.coolelectronics.me/breaking-cros-6/)

yes keep scrolling...

more of scrolling
|
ok almost there
|

ur almost there only 1 more part
|

use a 2000 psi pressure washer to crash the chromebook's components (you have taken of the back cover of the chromebook already) and buy a new chromebook. This is the most efficient way!

"Opting for a 2000 psi pressure washer presents a compelling advantage in cleaning power. Its higher pressure per square inch means it can effortlessly tackle tougher stains and grime, providing a more thorough and efficient cleaning experience. This increased pressure enables quicker removal of dirt and debris, making it particularly useful for larger areas or deeply embedded messes.

The 2000 psi model's superior force offers versatility, allowing it to handle a wider range of cleaning tasks effectively. From removing stubborn residues on outdoor surfaces to cleaning vehicles or patio furniture, the higher pressure provides a more comprehensive cleaning solution.

While the initial cost might be slightly higher compared to a 1500 psi washer, the enhanced performance and capability justify the investment, especially for those seeking a more robust and powerful cleaning tool. Additionally, the 2000 psi option's increased efficiency could potentially save time and effort during cleaning sessions.

Overall, the 2000 psi pressure washer stands out for its ability to swiftly and effectively handle various cleaning challenges, making it a preferable choice for those prioritizing power, efficiency, and versatility in their cleaning equipment."

**Reply by Hilololsusy** [reply]
Can I feel your hand option?

**Reply by Su4o** [reply]
noo. my |||||

**Reply by nathblade16** [reply]
I  have used this powerwashing meathod but my chromebook is now smoking, does anyone know a fix for this?

**Reply by Su4o** [reply]
oh, here's a simple easy tutorial!

Requirements:
Spatula
Grill
Chromebook
2000 psi pressure washer
land
electricity
2 hands. 
glasses to keep you safe

Just get a grill online and turn it one, you want to put it on high heat, _if you don't know how to just turn it 360._ 
Now once on, keep oil on the grill. _Get a spatula also.._ 
Then, you can keep the chromebook on the grill. It will be smoking but then, use your psi powerwash method again while the chromebook is on the grill.

 !solved profit

**Reply by nathblade16** [reply]
so my house lit on fire, how do I fix this

**Reply by Su4o** [reply]
Well there are 2 options on what you can do. 

OPTION A ---
1. get a new house next to the house on fire
2. now once the new house is on fire, use your psi pressure washer to spray water really hard onto both houses. (Note: if this is a class B type of fire, you're going to have to use option 2 instead)
3. now both houses are not one fire! get a new house and live in that one instead.

OPTION B ---
1. go to your chromebook that you have. [if you don't have one](https://www.google.com/chromebook
2. go onto to the chromebook and type in the url box "how do i get my house on fire again and burn it and get a house."
3. you should come across this [tutorial](https://www.hometowndemolitioncontractors.com/blog/burning-down-the-house-a-homeowners-guide-to-fire-department-training-exercises)
4. now burn your house, run away, and buy a new house on your phone. If you don't have one, try buying the Falcon Supernova iPhone 6 Pink Diamond. pretty cheap actually.
5. on the phone, dial 911, if you dont have a sim card well womp womp. Use the wifi in the burnt house!!!
6. tell them that your house is on fire

!double profit
